components/wso2is7.key.manager/src/main/java/org/wso2/is/client/WSO2IS7KeyManagerConstants.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
Change license

Reply:
Done in https://github.com/wso2-extensions/apim-km-wso2is/pull/128/commits/1f610b0caf85c6b4f30d7ac9953c28727bde50f6